styleguide/derek/_partials/_footer.html

```diff
@@ -0,0 +1,5 @@
+<!--[footernav]-->
+<nav class="">
```

Comment:
This should be migrated into `_footer.jade` as 100% Jade or into an EJS file to favor long-form HTML. Either way this kind of split isn't very desirable.